scripts/zkai_attention_kv_stwo_median_timing_gate.py

```
@@ -0,0 +1,685 @@
+        if summary[key] != expected:
+            raise StwoMedianTimingGateError(f"CLI {key} drift")
+    if require_exact_int(summary["runs_per_envelope"], "runs_per_envelope") != RUNS_PER_ENVELOPE:
+        raise StwoMedianTimingGateError("CLI run count drift")
+    if summary["clock"] != "std_time_instant_elapsed_as_micros":
+        raise StwoMedianTimingGateError("CLI clock drift")
+    if summary["safety"] != EXPECTED_SAFETY:
+        raise StwoMedianTimingGateError("CLI safety drift")
+    if tuple(summary["non_claims"]) != NON_CLAIMS:
+        raise StwoMedianTimingGateError("CLI non-claims drift")
+    if tuple(summary["validation_commands"])[:4] != VALIDATION_COMMANDS[:4]:
+        raise StwoMedianTimingGateError("CLI validation command drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)